.dev/scripts/generate_changelog.py

```diff
@@ -222,4 +226,9 @@ def generate_module_section(key, prs):
     for k in MODULES.keys():
         clog.extend(generate_module_section(k, selected_prs))
 
+    if args.with_excluded:
+        clog += ["\n### Miscellaneous PRs excluded from changelog\n"]
```

Comment:
Is it possible to be more uniform in adding '\n'? L221 and L225 don't add one on the end, just the beginning. Adding only on the end always (and adding empty newlines) makes more sense to me while reading.

Reply:
I see the point that this does not look uniform, but what's the alternative that results in the same output? Can you give me a patch?

Reply:
Heading needs an extra empty line around it. For that, we can use a simple class
```py
class Heading(list):
    def __init__(self, title_str: str):
            super().__init__(['\n', title_str, '\n'])
```

Reply:
Yes, but sometimes a heading comes after another heading. In such cases, we don't want a double empty line between them.

To be honest though, I think this is worth neither yours nor my time. This is just a helper script in `.dev/` folder.